mmv1/templates/terraform/examples/identity_platform_project_default_config.tf.erb

```diff
@@ -18,4 +18,26 @@ resource "google_identity_platform_project_default_config" "<%= ctx[:primary_res
             }
         }
     }
-}
\ No newline at end of file
+
+    blocking_functions {
```

Comment:
Due to https://github.com/hashicorp/terraform-provider-google/issues/13327, tests are currently disabled on this resource. In short, the initial apply (and later the destroy) that is done during tests will clear out other settings on the project.

I think there are two main directions you can go from here:
1. Keep the changes where they are, but keep in mind that this bug is unresolved, so your users may experience difficulties. You can attempt to resolve the bug if you choose, which would help make these new features more usable. For testing, you will need to turn off `skip_test` locally, and run the test in your own environment to ensure things are working.
2. Move the changes to `identityplatform/Config.yaml`, which appears to use the same endpoint but without this bug. For testing, you could probably remove the `skip_vcr: true` so that they run in the PR, otherwise you will need to run the test in your own environment.

Reply:
I'm not sure this was addressed either. It seems like you're choosing to keep the changes in this resource (cc @tylerg-dev who I see is a reviewer and also authored `identityplatform/Config.yaml`  https://github.com/GoogleCloudPlatform/magic-modules/pull/6587).

Either way, you will need to provide some sort of proof (ie. output) from running the acceptance tests.

Reply:
I can't see why we'd have a duplicate at all?  this one has more fields, but those should have just been merged into the initially added resource.  
At this point do we have a way to combine them back together (and properly use field-masks to avoid the bug listed above)

Reply:
Yea, the second resources was added with https://github.com/GoogleCloudPlatform/magic-modules/pull/6679, and I believe it was created due to a miscommunication combined with unfortunate timing. Since users could be using the newer resource, we cannot simply remove or merge it, but we could choose to start down the deprecation process and remove it in the next major release.

Ideally, I think the fields on this resource should be moved to yours, and this resource removed.